reactor-core/src/main/java/reactor/core/publisher/FluxPublishOn.java

```diff
@@ -1080,10 +1107,8 @@ public Object scanUnsafe(Attr key) {
 			if (key == Attr.PARENT) return s;
```

Comment:
above: `REQUESTED_FROM_DOWNSTREAM` should return `this.requested & Long.MAX_VALUE`, not `requested`, right?